<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ETC过车模拟器</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>256</width>
      <height>641</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>601</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRSU1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>启动天线一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRSU2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>启动天线二</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>一型客车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二型客车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三型客车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四型客车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>一型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>五型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>六型货车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>一型专项作业车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二型专项作业车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三型专项作业车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四型专项作业车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>五型专项作业车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>六型专项作业车</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>获取数据</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>日志</string>
     </attribute>
     <widget class="QPlainTextEdit" name="logArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>B2内容</string>
     </attribute>
     <widget class="QListView" name="lvB2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>B3内容</string>
     </attribute>
     <widget class="QListView" name="lvB3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>B4内容</string>
     </attribute>
     <widget class="QListView" name="lvB4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbParser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>解析数据帧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>110</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>连接IO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>320</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>自助车道模拟前天线过车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>自助车道模拟后天线过车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>416</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ETC专道模拟过车</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="cbRSU1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>天线一</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cbRSU2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>天线二</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbSender">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>70</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearIO</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rbEntry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string>入口</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbExit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>出口</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ETC专道模拟三车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>468</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ETC专道模拟两车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存选中</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>230</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>发送车牌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>180</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>连接车牌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>893</x>
      <y>230</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>发送RSU</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbParser_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>调整入口时间</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbParser_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>200</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭心跳</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_I"/>
    <addaction name="separator"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhello"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_I">
   <property name="text">
    <string>导入...(&amp;I)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="actionhello">
   <property name="text">
    <string>hello</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
